src/libs/actions/OnyxUpdateManager.js

```diff
@@ -35,6 +37,18 @@ export default () => {
                 return;
             }
 
+            // Since we used the same key that used to store another object, let's confirm that the current object is
+            // following the new format before we proceed. If it isn't, then let's clear the object in Onyx.
+            if (
+                !_.isObject(val) ||
+                !Object.prototype.hasOwnProperty.call(val, 'type') ||
```

Comment:
NAB: We could use `_.has` instead since we generally prefer underscore functions.

Reply:
Thanks, I didn't know that existed. Updated to it